### DISCUSSION TITLE
Component is being updated… How long should I expect it to take?

### DISCUSSION THREAD

**Discussion by keunes** [WeblateOrg/weblate#6761] [Q&A] [answered]
Hello,
I just set up a new project (AntennaPod website, libre trial) and added a component. However, I'm now stuck with this screen, and nothing seems happening:
![image](https://user-images.githubusercontent.com/11229646/140618526-f0ee6f9b-cf47-46ae-afc3-da1b8cbdbd16.png)

How long (roughly) should I expect it to take to import texts? Am I not supposed to see _something_? There seems to be a progress bar and a text output field, but nothing seems to be happening, so I'm wondering if something went wrong.

**Comment by keunes** [comment, marked as answer]
Ok, after 'adding a new translation' which was already partially translated in the repository, suddenly all the counts etc showed up, so that seems fine now. Still wondering if this screen I posted was supposed to show anything.

**Reply by nijel** [reply]
Yes, it should show a log of what is being processed. The hosted.weblate.org service had some performance issues during the weekend and the processing could be delayed for hours.

**Reply by nathanfallet** [reply]
I'm having the same problem today, with a weblate installation on my own server.

**Reply by nijel** [reply]
It can also be caused by Celery process not running.

**Reply by nathanfallet** [reply]
Yes the problem was that Celery didn't start. After, it worked perfectly
